integration/test_flow_integration.py

```diff
@@ -318,6 +318,38 @@ def test_write_to_v3io_stream_unbalanced(assign_stream_teardown_test):
     assert shard1_data == []
 
 
+@pytest.mark.parametrize('sharding_func_type', ['func', 'partition_num', 'field'])
+def test_write_to_v3io_stream_unbalanced(assign_stream_teardown_test, sharding_func_type):
+    if sharding_func_type == 'func':
+        sharding_func = lambda event: 0
```

Comment:
seems the coverage of the case where the function return the partition key is missing

Reply:
What do you mean? This line *is* the coverage.